modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -161,6 +161,136 @@ An additional role is required if the installation program is to create a vSpher
 |===
 ====
 
+.Roles and privileges required for installation in vCenter UI
```

Comment:
I missed here `in vCenter UI` to `in vCenter GUI`

Reply:
Addressed